### PR TITLE
LPS-64308 Implement toggleDisableInputs

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by jbalsas** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ToggleDisableInputs.es.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import Component from 'metal-component/src/Component';
+import core from 'metal/src/core';
+import Soy from 'metal-soy/src/Soy';
```
Comment (jbalsas) [inline modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ToggleDisableInputs.es.js]: This should be:

```
import Component from `metal-component`;
import core from `metal`;
import `Soy from `metal-soy`;
```

For this to work, the dependencies should be in installed in the `package.json`

File: modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import Component from 'metal-component/src/Component';
+import core from 'metal/src/core';
+import Soy from 'metal-soy/src/Soy';
```
Comment (jbalsas) [inline modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js]: This should be:

```
import Component from `metal-component`;
import core from `metal`;
import `Soy from `metal-soy`;
```

For this to work, the dependencies should be in installed in the `package.json`

File: modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	},
+
+	color: {
+		validator: core.isString
+	},
+
+	wrapperCssClass: {
+		validator: core.isString
+	}
+}
```
Comment (jbalsas) [inline modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js]: The component State is not documented

File: modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ToggleDisableInputs.es.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+	label: {
+		validator: core.isString
+	},
+
+	disableOnChecked: {
+		validator: core.isBoolean,
+		value: true
+	}
+}
```
Comment (jbalsas) [inline modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ToggleDisableInputs.es.js]: The component state is not documented

File: modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+					zIndex: Liferay.zIndex.POPOVER
+				}
+			);
+
+			instance.colorPickerPopover.render(instance.element);
+			instance.colorPickerPopover.after('select', instance.setColor_, instance);
+		});
+	}
+
+	setColor_(event) {
```
Comment (jbalsas) [inline modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js]: Missing jsdoc

File: modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ToggleDisableInputs.es.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
+import Component from 'metal-component/src/Component';
+import core from 'metal/src/core';
+import Soy from 'metal-soy/src/Soy';
+
+import templates from './ToggleDisableInputs.soy';
+
+class ToggleDisableInputs extends Component {
+	toggleSwitch() {
```
Comment (jbalsas) [inline modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ToggleDisableInputs.es.js]: Missing jsdoc

File: modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ToggleDisableInputs.es.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+		this.checked = !this.checked;
+
+		this.disableInputs_();
+	}
+
+	disableInputs_() {
```
Comment (jbalsas) [inline modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ToggleDisableInputs.es.js]: Missing jsdoc

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
@ealonso close, please

**Pull request closed without merging**